content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -218,10 +218,12 @@ sudo apt-get install cri-o-1.17
 
 {{< tab name="CentOS/RHEL 7.4+" codelang="bash" >}}
 # Install prerequisites
-yum-config-manager --add-repo=https://cbs.centos.org/repos/paas7-crio-115-release/x86_64/os/
+curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable.repo https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/CentOS_7/devel:kubic:libcontainers:stable.repo
+
+curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable:cri-o:[REQUIRED VERSION].repo https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:[REQUIRED VERSION]/CentOS_7/devel:kubic:libcontainers:stable:cri-o:[REQUIRED VERSION].repo
```

Comment:
What should the reader substitute for `[REQUIRED VERSION]`? (also, there might be a shortcode that can fill it in automatically during a website build).

Reply:
The substitute should be the crio version you would like to install(for example 1.17)
crio and Kubernetes follow the same release cycle and they share the same version number

Reply:
I think this will let the site build put the right version in automatically:
 `{{< skew latestVersion >}}`

For example:
```
curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable:cri-o:{{< skew latestVersion >}}.repo https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:{{< skew latestVersion >}}/CentOS_7/devel:kubic:libcontainers:stable:cri-o:{{< skew latestVersion >}}.repo
```